openedx/adg/lms/webinars/helpers.py

```diff
@@ -96,7 +99,7 @@ def send_webinar_registration_email(webinar, email):
         'webinar_title': webinar.title,
         'webinar_description': webinar.description,
         'webinar_link': webinar.meeting_link,
-        'webinar_start_time': webinar.start_time.strftime('%b %d, %Y %I:%M %p GMT'),
+        'webinar_start_time': webinar.start_time.astimezone(pytz.timezone(WEBINAR_DEFAULT_TIME_ZONE)).strftime('%b %d, %Y %I:%M %p GMT'),
```

Comment:
Can we please define a constant for this format too?

Reply:
Also why are we using two different formats at these two places? here and line number `36`

Reply:
The change was already there and i don't have any context about why the last dev didn't used the constant here as this change is in master . if you want i can make this change in context of my ticket too.

Reply:
Made the changes in each section where we are using the timezone and format